### PR TITLE
Add 180 minutes timeout to SLFO product building pipelines.

### PR DESCRIPTION
**Pull request by gleidi-suse**

### CODE REVIEW

File: gocd/slfo-stagings.gocd.yaml [hunk 1]
```diff
@@ -290,14 +304,21 @@ pipelines:
...
         tasks:
           - script: |-
+              minutes=$(expr 0)
               osc -A $STAGING_API api -X POST "/source/$STAGING_PROJECT?cmd=remove_flag&repository=product&flag=build"
               export PYTHONPATH=$PWD/scripts
               while ./scripts/gocd/verify-repo-built-successful.py -A $STAGING_API -p $STAGING_PROJECT -r product; ret=$?; [ ${ret} -ne 0 ]; do
                 if [ ${ret} -eq 2 ]; then
                     echo "product repository not found. Project configuration issue?" >&2
                     exit 1
+                elif [ ${ret} -ne 0 ] && ; then
```
Comment (DimStar77) [inline gocd/slfo-stagings.gocd.yaml:314]: `&&` is useless (if not even wrong) - there is no 2nd condition

Comment (DimStar77) [reply gocd/slfo-stagings.gocd.yaml:314]:
```
 if [ 0 -eq 0 ] && ; then
-bash: syntax error near unexpected token `;'
```

File: gocd/slfo-stagings.gocd.yaml.erb [hunk 2]
```diff
@@ -274,8 +281,14 @@ pipelines:
                 if [ ${ret} -eq 2 ]; then
                     echo "product repository not found. Project configuration issue?" >&2
                     exit 1
+                elif [ ${ret} -ne 0 ]; then
```
Comment (g7) [inline gocd/slfo-stagings.gocd.yaml.erb:284]: I think we can get rid of this and only keep the timeout check, as the exit status it's already checked before (line 280)

**Review by g7 — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**